resolwe_bio/processes/fusion_calling/arriba.py

```diff
@@ -0,0 +1,132 @@
+"""Gene fusion detection with Arriba."""
+
+from pathlib import Path
+from plumbum import TEE
+from resolwe.process import (
+    Cmd,
+    DataField,
+    FileField,
+    Process,
+    SchedulingClass,
+    StringField,
+)
+
+
+class Arriba(Process):
+    """Run Arriba for fusion detection from RNA-Seq data.
+
+    Arriba is a command-line tool for gene fusion detection from RNA-Seq data.
+    This process detects gene fusions from RNA-Seq data using the Arriba tool.
+    The input can be a BAM file from the STAR aligner, and additional optional
+    inputs such as blacklist, protein domains, and known fusions files can be provided.
+    More information about Arriba can be found in the
+    [Arriba manual](https://arriba.readthedocs.io/en/latest/) and in
+    the [original paper](https://genome.cshlp.org/content/31/3/448)
+    The current version of Arriba is 2.4.0.
+    """
+
+    slug = "arriba"
+    name = "Arriba"
+    process_type = "data:genefusions:arriba"
+    version = "1.0.0"
+    category = "RNA-Seq"
+    scheduling_class = SchedulingClass.BATCH
+    entity = {"type": "sample"}
+    requirements = {
+        "expression-engine": "jinja",
+        "executor": {
+            "docker": {"image": "public.ecr.aws/genialis/resolwebio/rnaseq:6.2.0"}
+        },
+        "resources": {
+            "cores": 4,
+            "memory": 16384,
+        },
+    }
+    data_name = '{{ bam|name|default("?") }}'
+
+    class Input:
+        """Input fields for Arriba process."""
+
+        bam = DataField("alignment:bam:star", label="Input BAM file from STAR aligner")
+        chim_out_type = StringField(
+            label="Chimeric output type",
+            description="Specify the chimeric output type.",
+            choices=["WithinBam", "SAMSeparateOld"],
+            default="WithinBam",
+        )
+        output_file = StringField(
```

Comment:
This can be omitted, as it is best to create output file name based on the sample slug, with `_fusions.tsv` suffix.